enterprise/cmd/frontend/internal/app/app.go

```diff
@@ -76,6 +77,16 @@ func Init(
 	return nil
 }
 
+func checkIfUserCanInstallGitHubApp(ctx context.Context, db database.DB, userID int32) error {
+	enabled, err := db.FeatureFlags().GetUserFlags(ctx, userID)
+	if err != nil {
+		return err
+	} else if !enabled["github-app-cloud"] {
+		return errors.New("Sourcegraph Cloud GitHub App setup is not enabled for the authenticated user")
```

Comment:
This sounds better to me, but feel free to leave it as is if you like your version better.

```suggestion
		return errors.New("Sourcegraph Cloud GitHub App setup is not enabled for current user")
```

Reply:
I don't feel strong about this as long as we have some guidance and consistency, unfortunately we have extensive uses of both ["the authenticated user"](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+the+authenticated+user&patternType=literal) and ["the current user"](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+the+current+user&patternType=literal) in our codebase. I'll keep as-is since the former would get my vote from technical perspective.

@quinnkeast: I think this is the type of terminology we should be consistent about throughout our product, have PDs had any discussions around this?